civihr_default_theme/scss/pages/_hr-details.scss

```diff
@@ -65,3 +65,198 @@
     }
   }
 }
+
+.view-mydetails-personal {
+  .contact-id span {
+    @include views-label;
+  }
+
+  .pane-block {
+    border-bottom: 1px solid $gray-light;
+    margin-bottom:0px;
+  }
+
+  .views-row {
+    min-height: 80px;
+    padding-left: 100px;
+    position: relative;
+  }
+
+  .views-field-image-URL {
+    left: 0px;
+    position: absolute;
+    top: 0px;
+  }
+
+  .no-user-image {
+    background: url(/sites/all/modules/civihr-custom/civihr_employee_portal/images/profile-default.png) center no-repeat;
+    background-size: 80px 80px;
+    overflow: hidden;
+    text-indent: -1000px;
+  }
+
+  .views-field-image-URL img,
+  .no-user-image {
+    border-radius: 50%;
+    height: 80px;
+    width: 80px;
+  }
+
+  .chr_panel__footer {
+    border-top: none;
+    margin-top: 0px;
+  }
+
+  .views-field-nothing .field-content {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-between;
+    white-space: nowrap;
+    width: 100%;
+    > * {
+      margin-right: 1rem;
+    }
+  }
+
+  .views-field-ni-ssn-70 .field-content {
+    width: 100%;
+  }
+
+  .nsi-exists.nsi-application-in-progress-1 {
+    > * {
+      display: none;
+    }
+
+    .nsi-application-status {
+      display: block;
+    }
+  }
+
+  div.views-field-display-name {
```

Comment:
why do we need to specify the `div`?

Reply:
A label and a value had the same class and I don't want to affect both, however I found another way to reference only the value.